### ISSUE TITLE
Set Arcanist Editor broken

### ISSUE THREAD

**Issue by lowlydba** [christianvuerings/vscode-phabricator#11]
> Command 'Phabricator: Set Arcanist editor to VS Code' resulted in an error (command 'phabricator-vscode.setAranistEditorToVscode' not found)

**Version:** 1.0.7
**VS Code Version:** 1.45.1

Unable to use the feature to set Arcanist editor as VS Code. Looks to be a typo based on the error message?

**Comment by christianvuerings** [comment]
@LowlyDBA thank you for flagging this issue. Could you share

* which OS + version you're using?
* the output of the Phabricator extension?
![image](https://user-images.githubusercontent.com/127199/83063003-57242c80-a014-11ea-9e33-47f07562a810.png)

I wasn't able to repro the bug locally with version 1.0.7 and VS Code 1.45.1. The function name does contain a typo but so did the call site.
![image](https://user-images.githubusercontent.com/127199/83062124-006a2300-a013-11ea-8932-d222d2e72371.png)

**Comment by christianvuerings** [comment]
@LowlyDBA you can also run the command in the terminal instead:

```
arc set-config editor "code --wait"
```

**Comment by lowlydba** [comment]
Looks like user error :) 

I had tried the "set editor" command first, _then_ set the API token, after which the command still did not work. It took a full restart of VS Code to get the "set editor" command working after configuring the settings.

Running the `arc` command in the terminal worked (and was my temporary workaround) until I discovered the above.

Mac OS: 10.14.6 (18G4032) - in case its relevant.
